taskfiles/k8s.yml

```diff
@@ -156,12 +156,23 @@ tasks:
         silent: true
       - 'echo "~~~ :docker: Build & Push operator container"'
       - docker buildx build
-        --platform linux/arm64,linux/amd64
+        --provenance false --sbom false --platform linux/amd64
         --file operator/Dockerfile
         --target manager
-        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
+        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}-amd64
         --push
```

Comment:
Do we want to remove the `--push` here if we're manually joining the two image tags below? Won't this wind up with us pushing 3 image tags?

Reply:
I'm not an expert in docker hub nor container registry, but if you are pushing manifest you are not uploading/sending/providing actual container layers. Sole manifest is useless without actual image, with its layers, in container registry.

Reply:
> I wonder if the docker manifest command might have a way to bundle the helm-chart into the same tag 🤔

It [actually seems likely](https://containers.gitbook.io/build-containers-the-hard-way#manifest) we could get this to work though I'm not sure if it's entirely worth the effort. It would just be awesome to bundle the chart into the image.

Reply:
I can test it, but my intuition tells me manifest (metadata) would not work.

Reply:
It doesn't work. I'm reverting.

https://buildkite.com/redpanda/redpanda-operator/builds/3948#019446d5-4701-446b-af7c-91a4d581ba92/344-437

As I said and you can see in the article https://containers.gitbook.io/build-containers-the-hard-way#manifest manifest is just a reference or I would say a metadata. If manifest can not find particular manifest in the registry it will stop you from creating a manifest.